src/Components/WebAssembly/WebAssembly.Authentication/src/Services/DeserializedAuthenticationStateProvider.cs

```diff
@@ -11,13 +11,13 @@ namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication;
 
 internal sealed class DeserializedAuthenticationStateProvider : AuthenticationStateProvider
```

Comment:
You're missing registrations of `DeserializedAuthenticationStateProvider` as PersistentComponentStateService. For reference, see how `DefaultAntiforgeryStateProvider` is registered on Server and WASM.